daphne/src/roles.rs

```diff
@@ -595,6 +645,11 @@ pub trait DapHelper<S>: DapAggregator<S> {
             return Err(DapAbort::UnauthorizedRequest);
         }
 
+        // Check whether the DAP version in the request is not Unknown.
```

Comment:
This should happen first (before authorization).

Reply:
Ditto :)